core/capabilities/ccip/ccipsolana/gas_helpers.go

```diff
@@ -1,14 +1,27 @@
 package ccipsolana
 
 import (
+	"fmt"
+
+	ccipcommon "github.com/smartcontractkit/chainlink/v2/core/capabilities/ccip/common"
+
 	cciptypes "github.com/smartcontractkit/chainlink-ccip/pkg/types/ccipocr3"
 )
 
-func NewGasEstimateProvider() EstimateProvider {
-	return EstimateProvider{}
+const (
+	// TODO: determine exact values
+	PerTokenOverheadGas = 75_000  // releaseOrMint using callWithExactGas
+	DestGasOverhead     = 350_000 // Commit and Exec costs
```

Comment:
Is there a plan for when the exact values will be determined? Or does that not matter for initial release?

Reply:
I think Matt had answers for these TODOs in the original PR. IIRC one of these isn't relevant anymore and the others have tickets to track them